app/boards/shields/nibble/nibble.overlay

```diff
@@ -47,3 +54,21 @@ RC(4,0) RC(4,1) RC(4,2) RC(4,3)                 RC(4,6)                 RC(4,9)
 		>;
 	};
 };
+
+&pro_micro_i2c {
+	status = "disabled";
```

Comment:
I believe you'll want this enabled to have `ZMK_DISPLAY=y` alone turn on the display for you

Reply:
The example in the nibble assembly docs doesn't include an OLED so it's disabled by default. I updated the readme to include a note about changing the status of the pro_micro_i2c to the readme in case folks are looking to enable the OLED instead of enabling it by default where it wouldn't be used by anything.

Reply:
I'm not sure I understand. That seems to be the point of ZMK_DISPLAY being off by default. Having this disabled as well just means you need to enable two separate things.

Reply:
This is a similar approach I took with the tidbit. I can enable the i2c side but i feel like it mildly complicates the builds w/o the need for i2c (and display).

Mind if I drag @petejohanson in for final verdict?